.pre-commit-config.yaml

```diff
@@ -0,0 +1,34 @@
+# configuration for pre-commit git hooks
```

Comment:
I really don't love the idea of black. I started a discussion #27 , please comment there.

Reply:
I removed it for now.